app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ActiveRecord::Base
+  include PublicActivity::Common
+  belongs_to :repository
+  belongs_to :author, class_name: "User", foreign_key: "user_id"
+
+  validates :body, presence: true
+
+  # Returns true if the user is the author of the comment
+  def author?(user)
+    author.id == user.id
```

Comment:
i would say it is better to compare using the `user_id` field, instead of building the author object which is maybe not needed.